src/app/features/test-records/components/base-test-record/base-test-record.component.ts

```diff
@@ -68,6 +68,10 @@ export class BaseTestRecordComponent implements AfterViewInit {
     return this.testRecordsService.sectionTemplates$;
   }
 
+  get editingTestResult$(): Observable<string | undefined> {
```

Comment:
Should probably be called something like `resultOfTest$` to match the rest of the application. Especially since `editingTestResult$` is something else in the component

Reply:
Thanks that change has been made.